text/0519-2019-2020-roadmap.md

```diff
@@ -0,0 +1,90 @@
+- **Modernize our build system.** This year, we will put the finishing touches on Embroider and make it the default build pipeline for new applications, unlocking optimizations like tree-shaking and route-based code splitting.
+- **Better a11y by default**. We will fix the router to work out-of-the-box with assistive technologies, and give developers more tools for understanding and improving the accessibility of their Ember applications.
+- **Optimize for growth.** A feeling of momentum is critical for keeping the Ember community healthy. We will double down on evangelizing Ember Octane and making Ember approachable to a new generation of web developers.
+
+### Continue simplifying Ember
+
+Ember Octane put us on a strong footing, modernizing the most foundational APIs in Ember. We will build on that foundation to dramatically simplify the framework, **reducing the overall number of concepts** so that Ember is easier to learn, use, and debug.
+
+We will **extract or deprecate functionality that is no longer widely needed**. With `get()` and `set()` mostly gone, so goes the need for many Ember-specific utility functions. Some utilities can be moved into optional packages for those who need them, while others may be better replaced by libraries like [Lodash](https://lodash.com/).
+
+Candidates for deprecation or extraction include:
+
+- **Object utilities** such as `isBlank`, `compare`, `trySet`, etc.
+- **String utilities** such as `camelize` and `dasherize`.
+- **Computed property macros** such as `alias`, `mapBy`, `notEmpty`, etc.
+- **Legacy object model features** such as observers, `setProperties`, `incrementProperty`, etc.
+
+Perhaps the longest-awaited simplification is **eliminating controllers** from the Ember programming model, which we will do this year. This will require us to find a new home for query parameters.
+
+In addition to making Ember easier to learn, these simplifications can also translate into improved performance. We will enable **slimmed down builds of Ember** that allow you to drop deprecated features you no longer use, reducing the asset size of modern Ember apps and incentivizing teams to move to the Octane programming model.
+
+Building on the simplicity of Glimmer components, we’ll also implement **template imports**, which make it explicit which helper or component is being invoked, and **named blocks**, which make it even easier to build ergonomic, reusable components.
+
+Lastly, we’ll **simplify and modernize Ember Data,** including:
+
+- **Easier on-ramp for new users** by eliminating the need for adapters and serializers for simple use cases based on `fetch()`.
+- **Increased velocity and stability** through improved, modularized internals.
+- **More flexibility** when overriding built-in behavior, by exposing low-level primitives as public API.
+
+### Modernize our build system
```

Comment:
I think this section should add `ember-engines` as well. Ember Engines is about packaging as well allowing for distributed development, testing, and packaging of logically-grouped pieces of an application.

And new build system will make ember engines more easy and stable removing most of [ember-asset-loader](https://github.com/ember-engines/ember-asset-loader) in favor of embroider. We should figure out how code-splitting/tree-shaking will work with engines and if this is planned to this roadmap.